examples/scopes/tekscope/get_acquisition_data.py

```diff
@@ -0,0 +1,48 @@
+"""An example script for connecting and configuring scope for acquisition."""
+from tm_devices import DeviceManager
+from tm_devices.drivers import MSO6B
+from tm_devices.helpers import PYVISA_PY_BACKEND
+
+with DeviceManager(verbose=True) as device_manager:
+    # Enable resetting the devices when connecting and closing
+    device_manager.setup_cleanup_enabled = True
+    device_manager.teardown_cleanup_enabled = True
+
+    # Use the PyVISA-py backend
+    device_manager.visa_library = PYVISA_PY_BACKEND
+
+    # Creating Scope driver object by providing ip address.
+    scope: MSO6B = device_manager.add_scope("127.0.0.1")  # pyright: ignore[reportGeneralTypeIssues]
+
+    # Make channel 1 ON
+    scope.commands.display.waveview1.ch[1].state.write("ON")
+
+    # Set channel 1 vertical scale to 10mV
+    scope.commands.ch[1].scale.write(10e-3)
+
+    # Set horizontal record length to 20000
+    scope.commands.horizontal.recordlength.write(20000)
+
+    # Set horizontal position to 100
+    scope.commands.horizontal.position.write(10)
+
+    #  Set trigger type to Edge
```

Comment:
There's two spaces in this comment

Reply:
Since this is just an example I am going to leave this for now.